docs/application/web/guides/machine-learning/trainer.md

```diff
@@ -0,0 +1,294 @@
+
+Configuration file structure is described in [the guide for Tizen Native API](https://docs.tizen.org/application/native/guides/machine-learning/machine-learning-training/#create-a-model-from-ini-formatted-file). This is a simplified configuration for the model created in the previous section.
+
+```ini
+[model]
+type = NeuralNetwork
+batch_size = 32
+epochs = 2
+
+[optimizer]
+type = sgd
+learning_rate = 0.01
+
+[layer_input0]
+type = input
+input_shape = 1:28:28
+
+[layer_conv2d1]
+type = conv2d
+filters = 32
+kernel_size = "3, 3"
+activation = relu
+
+[layer_pooling2d2]
+type = pooling2d
+pooling = max
+pool_size = "2, 2"
+
+[layer_flatten3]
+type = flatten
+
+[layer_fc4]
+type = fully_connected
+unit = 100
+activation = relu
+
+[layer_fc5]
+type = fully_connected
+unit = 10
+activation = softmax
+```
+
+To load the configuration use `createModel` function.
+
+```javascript
+var configFile = "documents/model.ini";
+var model = tizen.ml.trainer.createModel(configFile);
+```
+
+Compile the model.
+
+```javascript
+model.compile({
+    loss: "cross",
+});
+```
+
+## Training
+
+Connect the dataset to the model.
+
+```javascript
+model.setDataset(dataset);
+```
+
+Now the model is ready to train. To do that use `run` method.
+
+```javascript
+var options = {
+    epochs: 2,
+    batch_size: 32,
+};
+
+function successCallback() {
+    console.log("done");
+}
+
+function errorCallback(error) {
+    console.log(error);
+}
+
+model.run(options, successCallback, errorCallback);
+```
+
+## Saving trained model to a file
+
+The trained model can be saved to a file for the future use. Let's use the `FORMAT_INI_WITH_BIN` format, which will produce two files: an `ini` file with the network structure and a binary file with the weights. There are more formats which can be found in [API reference](https://docs.tizen.org/application/web/api/7.0/device_api/mobile/tizen/ml_trainer.html#Model::saveToFile). Remember to add a proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites).
+
+```javascript
+var modelFile = "documents/trained_model.ini";
+model.saveToFile(modelFile, "FORMAT_INI_WITH_BIN");
+```
+
+## Loading trained model from a file
+
+A trained model can be loaded from a file, both its structure and weights. If the model was saved using `FORMAT_INI_WITH_BIN` format, which produces two files, provide a path to the `ini` file. It contains a reference to the binary file with weights. It will be loaded during the compilation phase. Remember to add a proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites).
+
+```javascript
+var model = tizen.ml.trainer.createModel();
+var modelFile = "documents/trained_model.ini";
+model.load(modelFile, "FORMAT_INI_WITH_BIN");
+model.compile();
+```
+
+## Destroying the model
+
+If the model or its elements are not used anymore, they should be destroyed to avoid memory leaks. Disposing a model also destroys all objects associated with it (such as layers).
```

Comment:
code snippets should be preceded by a “colon”
```suggestion
If the model or its elements are not used anymore, they should be destroyed to avoid memory leaks. Disposing a model also destroys all objects associated with it (such as layers):
```